tests/Feature/Api/V1/AchievementSetClaimTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use LegacyApp\Community\Enums\ClaimSetType;
+use LegacyApp\Community\Enums\ClaimSpecial;
+use LegacyApp\Community\Enums\ClaimStatus;
+use LegacyApp\Community\Enums\ClaimType;
+use LegacyApp\Community\Models\AchievementSetClaim;
+use LegacyApp\Platform\Models\Game;
+use LegacyApp\Platform\Models\System;
+use Tests\TestCase;
+
+class AchievementSetClaimTest extends TestCase
+{
+    use RefreshDatabase;
+    use BootstrapsApiV1;
+
+    public function testGetActiveClaims(): void
+    {
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+        AchievementSetClaim::factory()->count(100)->create();
+
+        $response = $this->get($this->apiUrl('GetActiveClaims'))
+            ->assertSuccessful();
+
+        $this->assertCount(100, $response->json());
+    }
+
+    public function testGetUserClaims(): void
+    {
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+        /** @var AchievementSetClaim $claim */
+        $claim = AchievementSetClaim::factory()->create([
+            'GameID' => $game->ID,
+            'User' => $this->user->User,
+        ]);
+
+        $this->get($this->apiUrl('GetUserClaims', ['u' => $this->user->User]))
+            ->assertSuccessful()
+            ->assertExactJson([
+                [
+                    'ClaimType' => ClaimType::Primary,
```

Comment:
I would assume these asserts should use the expected integers, rather than the enums, to catch if the enums gets reordered. However, as the values are stored in the database, that seems unlikely.

Reply:
Yeah, a similar thing could be said about the point values etc in the achievement endpoints where the same value of the models is tested again which makes those not the most robust.
I'd say it's more important for now to check if the schema, the keys in particular, are present. That caused the biggest problem before (removal of the user ID) plus it seems with PHP 8.0 we cannot be too sure about the proper casting within the sqlite driver.
We can revisit this as soon as we're running on PHP 8.2 and be more strict about the actual values?